settings.gradle

```diff
@@ -146,3 +146,5 @@ includeWithFlags ':examples:spring-boot-webflux',                  'java'
 includeWithFlags ':examples:static-files',                         'java'
 includeWithFlags ':examples:thrift',                               'java'
 includeWithFlags ':examples:tutorials:rest-api-annotated-service', 'java'
+includeWithFlags ':examples:tutorials:grpc-tutorial',              'java'
```

Comment:
Should we rename this? 
`rest-api-annotated-service` does not contain `tutorial` in its name.

Reply:
Ah never mind. We have `grpc` module so we need a different name. :